internal/scaffold/ansible/usersetup.go

```diff
@@ -38,13 +38,11 @@ const userSetupTmpl = `#!/bin/sh
 set -x
 
 # ensure $HOME exists and is accessible by group 0 (we don't know what the runtime UID will be)
+echo "${USER_NAME}:x:${USER_UID}:0:${USER_NAME} user:${HOME}:/sbin/nologin" >> /etc/passwd
 mkdir -p ${HOME}/.ansible/tmp
```

Comment:
Could we keep or add a comment with the OCP ref to make clear why line 41 is required?